src/app/shared/services/student-profile.service.ts

```diff
@@ -57,16 +64,40 @@ export class StudentProfileService {
   getStudent(studentId: number): Observable<Option<Student>> {
     return this.loadingService.load(
       this.studentService
-        .get(studentId, {
-          context: new HttpContext().set(RestErrorInterceptorOptions, {
-            disableErrorHandlingForStatus: [404],
+        .get(studentId, { context: this.IGNORE_404_CONTEXT })
```

Comment:
Hier fänd' ich es noch schön, wenn es zwei Methoden gibt, z.B. `fetchStudent` und `fetchPerson` (beide mit Return-Type `Observable<Option<Student>>`) und dann hier in diesem `getStudent` nur die übergelagerte Logik mit dem Fallback drin wäre...

Reply:
Nice :ok_hand: